desktop/src/main/java/bisq/desktop/main/offer/takeoffer/TakeOfferViewModel.java

```diff
@@ -462,6 +463,7 @@ private void updateButtonDisableState() {
         boolean inputDataValid = isBtcInputValid(amount.get()).isValid
                 && dataModel.isMinAmountLessOrEqualAmount()
                 && !dataModel.isAmountLargerThanOfferAmount()
+                && dataModel.mempoolStatus.get() >= 0
```

Comment:
Initially I had it not blocking the next button, but depending on the speed of the mempool response you'd be told about a bad offer sometimes but not others.  So it felt unpredictable, and since the `checkOfferAvailabilityRequest` already blocks the next button I thought it best to have the mempool check do the same.